org.freecadweb.FreeCAD.yaml

```diff
@@ -602,3 +613,11 @@ modules:
           - sed -i 's/SHIBOKEN_MICRO_VERSION/0/g' src/Gui/WidgetFactory.cpp
           - find src ! -wholename "*/Mod/Sketcher/*" ! -wholename "*/Mod/PartDesign/*" -name "*.cpp" -type f -exec sed -i '1i using namespace std;' {} \;
           - sed -i '1i \#include <Inventor\/SbByteBuffer.h>' src/Gui/Quarter/QuarterWidget.cpp
+
+  - name: openscad-wrapper
```

Comment:
Can be merge into the shell source above.

Reply:
I meant keeping it above Will fix.

Reply:
I don't want to be pedantic, but the manifest is already 604 lines, please just merge this with the FreeCAD module!

Reply:
done